src/main/java/at/hannibal2/skyhanni/features/event/spook/TheGreatSpook.kt

```diff
@@ -53,11 +62,80 @@ object TheGreatSpook {
         if (isTimeLeftEnabled()) config.positionTimeLeft.renderString(displayTimeLeft, posLabel = "Time Left Display")
     }
 
+    private val mathFearMessagePattern by RepoPattern.pattern(
+        "chat.math",
+        "§d§lQUICK MATHS! §r§7Solve: §r§e(?<math>.*)",
+    )
+
+    private val speakingFearMessagePattern by RepoPattern.pattern(
+        "chat.speaking",
+        "§4\\[FEAR] Public Speaking Demon§r§f: (Speak|Say something interesting) (?<name>.*)!",
+    )
+
+    private fun solveMath(math: String): String {
+        val mathFormatted = math.replace("x", "*").replace(" ", "")
+        return try {
+            val result = MathUtils.evaluateExpression(mathFormatted)
+            result.toString()
+        } catch (e: Exception) {
+            "§c§lError"
```

Comment:
instead of saying error, return null and handle it custom

Reply:
I return Strings, how would I return null?

Reply:
You change the return value from `String` to `String?`. Then you can replace `"§c§lError"` with `null`. Then you can deal with the nullable in the caller place. (the other function where you were using the function´).